flathunter/chrome_wrapper.py

```diff
@@ -66,7 +66,7 @@ def get_chrome_driver(driver_arguments):
     chrome_version = get_chrome_version()
     # something is weird with the patched driver version (maybe only in python3.11), I had to patch
     # the chrome options to make it work
-    setattr(chrome_options, "headless", True)
+    # setattr(chrome_options, "headless", True)
```

Comment:
If this fixes an issue for you, that's great, but I would put this is a separate commit - seems unrelated to text generation.

Reply:
It fixed something for me, but I agree it's out of place here. Reverted.